bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/connector/PowerMaxSerialConnector.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax.internal.connector;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.util.TooManyListenersException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gnu.io.CommPort;
+import gnu.io.CommPortIdentifier;
+import gnu.io.NoSuchPortException;
+import gnu.io.PortInUseException;
+import gnu.io.SerialPort;
+import gnu.io.SerialPortEvent;
+import gnu.io.SerialPortEventListener;
+import gnu.io.UnsupportedCommOperationException;
+
+/**
+ * A class for the communication with the Visonic alarm panel through a serial connection
+ *
+ * @author lolodomo
+ * @since 1.9.0
+ */
+
+public class PowerMaxSerialConnector extends PowerMaxConnector {
+
+    private static final Logger logger = LoggerFactory.getLogger(PowerMaxSerialConnector.class);
+
+    private String serialPortName = "";
```

Comment:
no need to initialize to a value if only constructor sets it.  Also, if you change it nowhere but in the constructor, you could even make it final.